code/__DEFINES/~yogs_defines/flags.dm

```diff
@@ -0,0 +1,13 @@
+/*
+	These defines are specific to the atom/flags_1 bitmask
+*/
```

Comment:
You never include this file in the .dme?

Reply:
Fixed